index.js

```diff
@@ -50,7 +50,7 @@ async function updateTopTracks(json) {
   if (!tracks.length) return
 
   const lines = []
-  for (let index = 0; index < Math.min(tracks.length, 10); index++) {
+  for (let index = 0; index < Math.min(tracks.length, 50); index++) {
```

Comment:
maybe we should use a ENV like MAX_TRACKS, and default to 10, user can override by themself.

Reply:
I'm no good in JavaScript and because it's not related to this PR, I changed it back to 10 - also because 50 doesn't seem to make any sense.